bayesian/api_v1.py

```diff
@@ -583,6 +585,32 @@ def get_component_analysis_schema_url(cls, name, version):
                                    name=name, version=version)
 
 
+class GenerateManifest(Resource):
+    method_decorators = [login_required]
+
+    @staticmethod
+    def post():
+        input_json = request.get_json()
+        if 'ecosystem' not in input_json:
+            raise HTTPError(400, "Must provide a ecosystem")
+        if input_json.get('ecosystem') == 'maven':
+            return Response(
+                PomXMLTemplate(input_json).xml_file(),
```

Comment:
Method name `xml_file` is misleading. Probably something like xml_string() would be more accurate.